baselines/dom.generated.d.ts

```diff
@@ -3612,6 +3617,7 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     webkitMatchesSelector(selectors: string): boolean;
     webkitRequestFullscreen(): void;
     webkitRequestFullScreen(): void;
+    cloneNode(deep?: boolean): this;
```

Comment:
do not use `this` here. this makes everything that extends `HTMLElement` generic.